src/ray/gcs/gcs_server/BUILD.bazel

```diff
@@ -197,6 +197,7 @@ ray_cc_library(
         "//src/ray/rpc:core_worker_client",
         "//src/ray/rpc:gcs_server",
         "//src/ray/stats:stats_metric",
+        "//src/ray/telemetry:ray_job_event_recorder",
```

Comment:
the job manager should depend on a `job_event_recorder_interface`, not the concrete implementation

only the "main" file (in this case, `gcs_server.cc`) should depend on the concrete implementation